component/org.wso2.carbon.identity.api.otp.service.smsOtp/pom.xml

```diff
@@ -0,0 +1,223 @@
+            <!--                                <modelPackage>org.wso2.identity.api.otp.service.smsotp.dto</modelPackage>-->
+            <!--                                <packageName>org.wso2.identity.api.otp.service.smsotp</packageName>-->
+            <!--                                <dateLibrary>java8</dateLibrary>-->
+            <!--                                <hideGenerationTimestamp>true</hideGenerationTimestamp>-->
+            <!--                                <ignoreFileOverride>${project.basedir}/.openapi-generator-ignore</ignoreFileOverride>-->
+            <!--                            </configOptions>-->
+            <!--                            <output>.</output>-->
+            <!--                            <skipOverwrite>false</skipOverwrite>-->
+            <!--                        </configuration>-->
+            <!--                    </execution>-->
+            <!--                </executions>-->
+            <!--                <dependencies>-->
+            <!--                    <dependency>-->
+            <!--                        <groupId>org.openapitools</groupId>-->
+            <!--                        <artifactId>cxf-wso2-openapi-generator</artifactId>-->
+            <!--                        <version>1.0.0</version>-->
+            <!--                    </dependency>-->
+            <!--                </dependencies>-->
+            <!--            </plugin>-->
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>build-helper-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>add-source</id>
+                        <phase>generate-sources</phase>
+                        <goals>
+                            <goal>add-source</goal>
+                        </goals>
+                        <configuration>
+                            <sources>
+                                <source>src/gen/java</source>
+                            </sources>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-war-plugin</artifactId>
+                <configuration>
+                    <webResources>
+                        <resource>
+                            <!-- this is relative to the pom.xml directory -->
+                            <directory>src/main/webapp</directory>
+                        </resource>
+                    </webResources>
+                    <warName>api#identity#sms-otp#v1</warName>
+                    <webXml>src/main/webapp/WEB-INF/web.xml</webXml>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+</project>
```

Comment:
Leave a newline at the end of each file. Here and everywhere else.

Reply:
fixed in 69f620fe512912d50eddd5ca499c62117c1832ba